package.json

```diff
@@ -58,14 +58,15 @@
     "wkx": "^0.5.0"
   },
   "devDependencies": {
-    "@rushstack/eslint-patch": "1.1.0",
```

Comment:
I can remember other PRs that also change this and ibm_db so we don't have to do this here. I'll check which one and if that one is not looking like it will be merged soon I will make a separate PR